packages/next/build/babel/loader/get-config.ts

```diff
@@ -0,0 +1,198 @@
+import { default as nextBabelPreset } from 'next/babel'
+import { createConfigItem, loadOptions } from '@babel/core'
+import loadConfig from '@babel/core/lib/config'
+
+import type {
```

Comment:
Unfortunately we can't use `import type` because of backwards compat with earlier typescript versions users are using.

```suggestion
import {
```

Reply:
That's good to know!